apps/src/manual_registration/manual_registration.cpp

```diff
@@ -45,6 +45,10 @@
 #include <QEvent>
 #include <QMutexLocker>
 #include <QObject>
+#if VTK_MAJOR_VERSION >= 8 && VTK_MINOR_VERSION >= 2
+#define HAS_VTK_82
```

Comment:
Minor nitpick, but since you check for version greater than 8, this might not be entirely true, since you might have 9.0 😄
Maybe just use the same #if or rename it to, HAS_QVTKOPENGLWINDOW_H ? (oh ,last part look ugly 😁 )

Reply:
What you say @mvieth ?

Reply:
guess i shoulda named it `HAS_VTK_ATLEAST_82` …